pkg/cli/alpha/config-gen/cmd.go

```diff
@@ -124,7 +125,7 @@ func NewCommand() *cobra.Command {
 	c.Version = `v0.1.0`
 	c.Short = `Generate configuration for controller-runtime based projects`
 	c.Long = strings.TrimSpace(`
-config-gen programatically generates configuration for a controller-runtime based
+config-gen programmatically generates configuration for a controller-runtime based
```

Comment:
Could you please push a PR only for this change. 
That seems out of the scope of what you are looking for to introduce in this PR